tests/testthat/test-backend-mssql.R

```diff
@@ -164,9 +164,30 @@ FROM `df`")
 test_that("ORDER BY in subqueries uses TOP 100 PERCENT (#175)", {
   mf <- lazy_frame(x = 1:3, con = simulate_mssql())
 
-  expect_equal(
-    mf %>% mutate(x = -x) %>% arrange(x) %>% mutate(x = -x) %>% sql_render(),
-    sql("SELECT -`x` AS `x`\nFROM (SELECT TOP 100 PERCENT *\nFROM (SELECT TOP 100 PERCENT -`x` AS `x`\nFROM `df`) `dbplyr_001`\nORDER BY `x`) `dbplyr_002`")
+  rendered_query <-
+    mf %>%
+    mutate(x = -x) %>%
+    arrange(x) %>%
+    mutate(x = -x) %>%
+    sql_render()
+
+  expect_match(
```

Comment:
Needs to use `verify_output()`

Reply:
We settled on the following structure:

```r
test_that("...", {
  local_options(dbplyr_table_num = 0)
  verify_output(test_path("sql/<filename>.txt"), {
     ...
  }
}
```

The `local_options()` is important to keep the names of subqueries stable. Can you please update only this test as part of this pull request?

Reply:
Done in f1e0f47, can you cherry-pick from there?